src/App.vue

```diff
@@ -540,6 +428,59 @@ export default class App extends Mixins(
     }
   }
 
+  /** Check authorizations. */
+  async checkAuth (): Promise<void> {
+    try {
+      // load account information
+      await this.loadAccountInformation().catch(error => {
+        console.log('Account info error = ', error) // eslint-disable-line no-console
+        throw error
+      })
+
+      // safety check
+      if (!this.businessId) {
+        throw new Error('Missing Business ID')
+      }
+      // check if current user is authorized
+      const response = await AuthServices.fetchAuthorizations(
+        this.getAuthApiUrl, this.businessId
+      )
+      this.storeAuthorizations(response) // throws if no role
```

Comment:
Why do we need this as well as line 462? I think it's redundant.

Reply:
Yeah, storeAuthorizations is doing the same thing as the code below. I think you can remove it.